{
  "post_stream": {
    "posts": [
      {
        "id": 4411,
        "name": "",
        "username": "pragawaran",
        "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/p/f07891/{size}.png",
        "created_at": "2019-06-17T12:33:04.629Z",
        "cooked": "<p>When I run fdbbackup (generated binary using GitHub source code), I am facing this issue \u201cThe backup on tag `default\u2019 was successfully submitted but no backup agents are responding\u201d. How to resolve this issue\u2026?</p>",
        "post_number": 1,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-06-17T12:33:04.629Z",
        "reply_count": 0,
        "reply_to_post_number": null,
        "quote_count": 0,
        "incoming_link_count": 180,
        "reads": 61,
        "readers_count": 60,
        "score": 912.2,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": "",
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 587,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/1",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null,
        "can_vote": false
      },
      {
        "id": 4417,
        "name": "A.J. Beamon",
        "username": "ajbeamon",
        "avatar_template": "/user_avatar/forums.foundationdb.org/ajbeamon/{size}/13_2.png",
        "created_at": "2019-06-17T16:24:18.440Z",
        "cooked": "<p>Running fdbbackup starts the backup job, but it\u2019s the agents that are responsible for actually doing the work. You\u2019ll need to have at least one running for backup to actually make any progress. Depending on the size of your cluster and how quickly you want backup to go, you may want to run more than that.</p>\n<p>If you are using fdbmonitor to run your processes, these can be started by configuring them in foundationdb.conf, like so:</p>\n<pre><code class=\"lang-auto\">command = /usr/lib/foundationdb/backup_agent/backup_agent\nlogdir = /var/log/foundationdb\n\n[backup_agent.1]\n</code></pre>\n<p>If you are running your processes through some other mechanism, then you just need to start the agent processes with the cluster file for your cluster. See <a href=\"https://apple.github.io/foundationdb/backups.html?highlight=backup#backup-agent-command-line-tool\" rel=\"nofollow noopener\">https://apple.github.io/foundationdb/backups.html?highlight=backup#backup-agent-command-line-tool</a> for more details.</p>",
        "post_number": 2,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-06-17T16:24:18.440Z",
        "reply_count": 1,
        "reply_to_post_number": null,
        "quote_count": 0,
        "incoming_link_count": 7,
        "reads": 60,
        "readers_count": 59,
        "score": 52.0,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "A.J. Beamon",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "link_counts": [
          {
            "url": "https://apple.github.io/foundationdb/backups.html?highlight=backup#backup-agent-command-line-tool",
            "internal": false,
            "reflection": false,
            "title": "Backup, Restore, and Replication for Disaster Recovery \u2014 FoundationDB 6.1",
            "clicks": 14
          }
        ],
        "read": true,
        "user_title": null,
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 12,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/2",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5828,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-16T11:11:11.626Z",
        "cooked": "<aside class=\"quote no-group\" data-username=\"ajbeamon\" data-post=\"2\" data-topic=\"1458\">\n<div class=\"title\">\n<div class=\"quote-controls\"></div>\n<img loading=\"lazy\" alt=\"\" width=\"24\" height=\"24\" src=\"https://sea1.discourse-cdn.com/foundationdb/user_avatar/forums.foundationdb.org/ajbeamon/48/13_2.png\" class=\"avatar\"> ajbeamon:</div>\n<blockquote>\n<p>If you are using fdbmonitor to run your processes, these can be started by configuring them in foundationdb.conf, like so:</p>\n</blockquote>\n</aside>\n<p>hey <a class=\"mention\" href=\"/u/ajbeamon\">@ajbeamon</a><br>\nhow do I verify that I am running the processes using fdbmonitor. I installed the cluster using the foundationDB operator documented <a href=\"https://github.com/foundationdb/fdb-kubernetes-operator\" rel=\"noopener nofollow ugc\">here</a>.<br>\nA part from that the foundationDB documentation says, in the link that you shared, that we dont need to start backup_agent manually it usually runs automatically on the machine.</p>",
        "post_number": 3,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-16T11:11:11.626Z",
        "reply_count": 2,
        "reply_to_post_number": 2,
        "quote_count": 1,
        "incoming_link_count": 0,
        "reads": 53,
        "readers_count": 52,
        "score": 20.6,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "link_counts": [
          {
            "url": "https://github.com/foundationdb/fdb-kubernetes-operator",
            "internal": false,
            "reflection": false,
            "title": "GitHub - FoundationDB/fdb-kubernetes-operator: A kubernetes operator for FoundationDB",
            "clicks": 3
          }
        ],
        "read": true,
        "user_title": null,
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/3",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5830,
        "name": "gaurav",
        "username": "gaurav",
        "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/g/b487fb/{size}.png",
        "created_at": "2019-12-16T11:56:59.833Z",
        "cooked": "<p>You can do <code>ps -ef | grep fdb</code> and the verify if fdbmonitor is running and if its pid is same as fdbserver process parent pid.</p>\n<p>somehting like this:</p>\n<pre><code class=\"lang-auto\">ubuntu    1087     1  0 Dec12 ?        00:00:00 /usr/bin/fdbmonitor --conffile /etc/foundationdb/foundationdb.conf --lockfile /var/run/fdb/fdbmonitor.pid --daemonize\nubuntu    1099  1087  0 Dec12 ?        00:18:54 /usr/lib/foundationdb/backup_agent/backup_agent --cluster_file /etc/foundationdb/fdb.cluster --logdir /var/log/foundationdb\nubuntu    1100  1087 10 Dec12 ?        09:29:54 /usr/bin/fdbserver --cluster_file /etc/foundationdb/fdb.cluster --datadir /var/lib/foundationdb/data/4500 --listen_address public --logdir /var/log/foundationdb --public_address auto:4500</code></pre>",
        "post_number": 4,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-16T11:56:59.833Z",
        "reply_count": 1,
        "reply_to_post_number": 3,
        "quote_count": 0,
        "incoming_link_count": 3,
        "reads": 52,
        "readers_count": 51,
        "score": 30.4,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "gaurav",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 166,
        "hidden": false,
        "trust_level": 3,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/4",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5831,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-16T12:00:12.959Z",
        "cooked": "<p>Hi <a class=\"mention\" href=\"/u/gaurav\">@gaurav</a><br>\nBelow is the output  <code>ps -ef</code> in my case</p>\n<pre><code class=\"lang-auto\"> ps -ef\nUID        PID  PPID  C STIME TTY          TIME CMD\nroot         1     0  0 06:58 ?        00:00:00 sh -c fdbmonitor --conffile /var/dynamic-conf/fdbmonitor.conf --lockfile /var/fdb/fdbmonitor.lockfile\nroot         6     1  0 06:58 ?        00:00:00 fdbmonitor --conffile /var/dynamic-conf/fdbmonitor.conf --lockfile /var/fdb/fdbmonitor.lockfile\nroot         7     6  2 06:58 ?        00:06:04 /var/dynamic-conf/bin/6.2.11/fdbserver --class storage --cluster_file /var/fdb/data/fdb.cluster --datadir /var/fdb/data --knob_disable_posix_kernel_aio 1 --locality_instance_id 1 --locali\nroot       122     0  0 08:27 pts/0    00:00:00 bash\n\n</code></pre>\n<p>so it seems I have <code>fdbmonitor</code> running and its <code>PID</code> is  same as the <code>fdbserver</code>'s <code>PPID</code>.</p>",
        "post_number": 5,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-16T12:34:48.014Z",
        "reply_count": 1,
        "reply_to_post_number": 4,
        "quote_count": 0,
        "incoming_link_count": 1,
        "reads": 57,
        "readers_count": 56,
        "score": 21.4,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 2,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 166,
          "username": "gaurav",
          "name": "gaurav",
          "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/g/b487fb/{size}.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/5",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5832,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-16T13:52:48.106Z",
        "cooked": "<p>To resolve it I changed the <code>fdbmonitor.conf</code> file to have details of backup_agent, because it seems the backup_agent is not being started by default. So insert below snipped in your <code>fdbmonitor.conf</code> file.</p>\n<pre><code class=\"lang-auto\">[backup_agent]\ncommand = /usr/bin/backup_agent -C &lt;fdb-cluster-file &gt; \n\n[backup_agent.1]\n\n</code></pre>\n<p>in my case <code>fdb-cluster-file</code> is at  <code>/var/dynamic-conf/fdb.cluster</code>. After that you dont have to restart the service because this dir is always listened and if anything is changed the processes will be loaded accordingly. After doing the mentioned changes I was able to take the backup successfully.</p>",
        "post_number": 6,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-16T13:52:48.106Z",
        "reply_count": 0,
        "reply_to_post_number": 5,
        "quote_count": 0,
        "incoming_link_count": 91,
        "reads": 51,
        "readers_count": 50,
        "score": 465.2,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "link_counts": [
          {
            "url": "https://forums.foundationdb.org/t/backing-up-the-database-using-fdbbackup-command-returns-in-an-error/1844/2",
            "internal": true,
            "reflection": true,
            "title": "Backing up the database using `fdbbackup` command returns in an error",
            "clicks": 1
          }
        ],
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/6",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5835,
        "name": "John Brownlee",
        "username": "john_brownlee",
        "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
        "created_at": "2019-12-16T15:23:43.917Z",
        "cooked": "<p>When you\u2019re running FDB through the Kubernetes Operator, it doesn\u2019t start backup agents. You can start the backup agents through a deployment and have them get the cluster file from the config map that the operator creates. We should an example of this pattern to the documentation.</p>\n<p>We should also consider integrating backup management into the operator.</p>",
        "post_number": 7,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-16T15:23:43.917Z",
        "reply_count": 2,
        "reply_to_post_number": 3,
        "quote_count": 0,
        "incoming_link_count": 142,
        "reads": 50,
        "readers_count": 49,
        "score": 745.0,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "John Brownlee",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [
          {
            "id": 2,
            "count": 1
          }
        ],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 3,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/7",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5837,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-16T15:42:36.970Z",
        "cooked": "<p>Hey <a class=\"mention\" href=\"/u/john_brownlee\">@john_brownlee</a><br>\nthat makes sense but I was able to take backup somehow, as mentioned in my above post, but the problem now is I am getting some issues while restoring that backup. I have created a post about that, can you please look into that when you have time.</p><aside class=\"quote quote-modified\" data-post=\"1\" data-topic=\"1845\">\n  <div class=\"title\">\n    <div class=\"quote-controls\"></div>\n    <img loading=\"lazy\" alt=\"\" width=\"24\" height=\"24\" src=\"https://sea1.discourse-cdn.com/foundationdb/user_avatar/forums.foundationdb.org/viveksinghggits/48/737_2.png\" class=\"avatar\">\n    <a href=\"https://forums.foundationdb.org/t/restoring-a-completed-backup-version-results-in-an-error/1845\">Restoring a completed backup version results in an error</a> <a class=\"badge-category__wrapper \" href=\"/c/using-foundationdb/7\"><span data-category-id=\"7\" data-drop-close=\"true\" class=\"badge-category \" title=\"Topics related to use of FoundationDB.\"><span class=\"badge-category__name\">Using FoundationDB</span></span></a>\n  </div>\n  <blockquote>\n    I have created backup of the foundationDB successfully, if I check the status of the backup below is what I get \nThe previous backup on tag `default' at file:///data/fdbbackup/backup-2019-12-16-12-43-45.243778 completed at version 21655918762.\nBackupUID: 4b6d2cf2e97bd4a673b32a008ddaa17d\nBackupURL: file:///data/fdbbackup/backup-2019-12-16-12-43-45.243778\n\n\nBut When I try to restore the same backup using below command fdbrestore start --dest_cluster_file /var/dynamic-conf/fdb.cluster -r file:///da\u2026\n  </blockquote>\n</aside>\n",
        "post_number": 8,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-16T15:42:36.970Z",
        "reply_count": 0,
        "reply_to_post_number": 7,
        "quote_count": 0,
        "incoming_link_count": 0,
        "reads": 49,
        "readers_count": 48,
        "score": 9.8,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "link_counts": [
          {
            "url": "https://forums.foundationdb.org/t/restoring-a-completed-backup-version-results-in-an-error/1845",
            "internal": true,
            "reflection": false,
            "title": "Restoring a completed backup version results in an error",
            "clicks": 0
          }
        ],
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 3,
          "username": "john_brownlee",
          "name": "John Brownlee",
          "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/8",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5848,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-17T10:53:11.000Z",
        "cooked": "<p>Hi <a class=\"mention\" href=\"/u/john_brownlee\">@john_brownlee</a><br>\nI will bother you once again about this, I started experiencing the same issue once again today, and when I do the exactly same thing that I have mentioned above about changing the fdbmonitor.config file to add backup_agent section, things dont work. But this worked as expected yesterday. When I change the file with below content and same it</p>\n<pre><code class=\"lang-auto\">[backup_agent]\ncommand = /usr/bin/backup_agent -C &lt;fdb-cluster-file &gt; \n\n[backup_agent.1]\n</code></pre>\n<p>below is the output that i get in the pod\u2019s logs</p>\n<pre><code class=\"lang-auto\">Time=\"1576579785.791398\" Severity=\"10\" LogGroup=\"default\" Process=\"fdbmonitor\": Watching conf file /var/dynamic-conf/fdbmonitor.conf\nTime=\"1576579785.791595\" Severity=\"10\" LogGroup=\"default\" Process=\"fdbmonitor\": Watching conf dir /var/dynamic-conf/ (18)\nTime=\"1576579785.791615\" Severity=\"10\" LogGroup=\"default\" Process=\"fdbmonitor\": Loading configuration /var/dynamic-conf/fdbmonitor.conf\nTime=\"1576579785.791903\" Severity=\"10\" LogGroup=\"default\" Process=\"fdbmonitor\": Updated configuration for fdbserver.1\n\n</code></pre>\n<p>Can you please suggest how to about taking the backup of the database.</p>",
        "post_number": 9,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T10:53:11.000Z",
        "reply_count": 1,
        "reply_to_post_number": 7,
        "quote_count": 0,
        "incoming_link_count": 2,
        "reads": 40,
        "readers_count": 39,
        "score": 23.0,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 3,
          "username": "john_brownlee",
          "name": "John Brownlee",
          "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/9",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5851,
        "name": "John Brownlee",
        "username": "john_brownlee",
        "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
        "created_at": "2019-12-17T15:07:19.846Z",
        "cooked": "<p>In general, I would recommend running the backup agents through a Deployment rather than through fdbmonitor.</p>",
        "post_number": 10,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T15:07:19.846Z",
        "reply_count": 1,
        "reply_to_post_number": 9,
        "quote_count": 0,
        "incoming_link_count": 0,
        "reads": 38,
        "readers_count": 37,
        "score": 12.6,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "John Brownlee",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 3,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/10",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5852,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-17T16:18:49.737Z",
        "cooked": "<p>yeah, but there also even if we mount the configmap to the new backup_agent deployment and then provide the fdb cluster file using -r flag, do you think fdbbackup will be able to backup the data from the foundationdb pod, have you tested that.<br>\nOne more thing, if  my database is running through the operator pods and I want to run <code>fdbbackup</code> for the remote database, I am not on the same pod where my cluster file and other configuration are, how can we achieve that. Is there a way to provide a flag to <code>fdbbackup</code> so that it will take backup of the remote database.</p>",
        "post_number": 11,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T16:18:49.737Z",
        "reply_count": 1,
        "reply_to_post_number": 10,
        "quote_count": 0,
        "incoming_link_count": 1,
        "reads": 37,
        "readers_count": 36,
        "score": 17.4,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 3,
          "username": "john_brownlee",
          "name": "John Brownlee",
          "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/11",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5855,
        "name": "A.J. Beamon",
        "username": "ajbeamon",
        "avatar_template": "/user_avatar/forums.foundationdb.org/ajbeamon/{size}/13_2.png",
        "created_at": "2019-12-17T17:30:29.788Z",
        "cooked": "<p>Also as a side note, it shouldn\u2019t be necessary to specify the cluster file in the <code>command</code> part of your foundationdb.conf file. If the correct cluster file is listed in the <code>general</code> section, I think it will automatically get picked up by the process. If not, you should be able to specify it in the <code>backup</code> section in the same way:</p>\n<pre><code class=\"lang-auto\">[backup_agent]\ncommand = /usr/bin/backup_agent\ncluster_file = &lt;path_to_cluster_file&gt;\n</code></pre>",
        "post_number": 12,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T17:30:29.788Z",
        "reply_count": 0,
        "reply_to_post_number": null,
        "quote_count": 0,
        "incoming_link_count": 5,
        "reads": 37,
        "readers_count": 36,
        "score": 32.4,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "A.J. Beamon",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 12,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/12",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5857,
        "name": "John Brownlee",
        "username": "john_brownlee",
        "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
        "created_at": "2019-12-17T18:55:33.842Z",
        "cooked": "<p>The backup agents get their data by connecting to the cluster, rather than reading off of a disk, so they do not need to be in the same pod or on the same machine as the fdbserver processes. All you need to provide to backup_agent and fdbbackup is the path to the cluster file.</p>",
        "post_number": 13,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T18:55:56.973Z",
        "reply_count": 1,
        "reply_to_post_number": 11,
        "quote_count": 0,
        "incoming_link_count": 0,
        "reads": 35,
        "readers_count": 34,
        "score": 12.0,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "John Brownlee",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 3,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/13",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5859,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-17T19:14:27.887Z",
        "cooked": "<p>Yeah, I understood that but the problem is lets say if I run below command to run the backup_agent on the other pod</p>\n<pre><code class=\"lang-auto\">/usr/bin/backup_agent -C /var/dynamic-conf/fdb.cluster\n</code></pre>\n<p>here <code>-C</code> provides the mechanism to pass the cluster problem, now the confusion that I have is, where is <code>backup_agent</code> going to look for the file <code>/var/dynamic-conf/fdb.cluster</code>. Because backup_agent will not be able to find the file where it (<code>backup_agent</code>) is running.</p>",
        "post_number": 14,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T19:14:27.887Z",
        "reply_count": 1,
        "reply_to_post_number": 13,
        "quote_count": 0,
        "incoming_link_count": 1,
        "reads": 36,
        "readers_count": 35,
        "score": 17.2,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 3,
          "username": "john_brownlee",
          "name": "John Brownlee",
          "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/14",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5860,
        "name": "John Brownlee",
        "username": "john_brownlee",
        "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
        "created_at": "2019-12-17T19:23:20.168Z",
        "cooked": "<p>You will need to mount the cluster file into the container where the backup agent is running. You should be able to mount the same config map that the operator creates for the fdbserver processes, and get the cluster file from there.</p>",
        "post_number": 15,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T19:23:20.168Z",
        "reply_count": 1,
        "reply_to_post_number": 14,
        "quote_count": 0,
        "incoming_link_count": 2,
        "reads": 33,
        "readers_count": 32,
        "score": 21.6,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "John Brownlee",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 3,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/15",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5861,
        "name": "Vivek Singh",
        "username": "viveksinghggits",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "created_at": "2019-12-17T19:27:38.529Z",
        "cooked": "<p>Hi <a class=\"mention\" href=\"/u/john_brownlee\">@john_brownlee</a><br>\nI got that when we first discussed about that for the first time, but after looking into that configmap I had some doubts about how are we going to figure out where the cluster actually is. But when I looked into that file again, I think below is how backup_agent is going to figure out the cluster. Thanks</p>\n<pre><code class=\"lang-auto\">cluster-file: |\n    # DO NOT EDIT!\n    # This file is auto-generated, it is not to be edited by hand\n    foundationdbcluster_sample:VPnLf5797TLX6qSQeACWSSdt6k7OHYGO@10.244.0.84:4500:tls,10.244.0.90:4500:tls,10.244.0.91:4500:tls\n  \n</code></pre>\n<p>Can you suggest the binaries that should be there in the other pod that we are going to spin up for the backup_agent or any documentation link would help.</p>",
        "post_number": 16,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T19:28:45.717Z",
        "reply_count": 1,
        "reply_to_post_number": 15,
        "quote_count": 0,
        "incoming_link_count": 7,
        "reads": 33,
        "readers_count": 32,
        "score": 46.6,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "Vivek Singh",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 2,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 3,
          "username": "john_brownlee",
          "name": "John Brownlee",
          "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": false,
        "staff": false,
        "user_id": 700,
        "hidden": false,
        "trust_level": 1,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/16",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      },
      {
        "id": 5863,
        "name": "John Brownlee",
        "username": "john_brownlee",
        "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
        "created_at": "2019-12-17T19:51:11.604Z",
        "cooked": "<p>We have more documentation on backup in general here: <a href=\"https://apple.github.io/foundationdb/backups.html\">https://apple.github.io/foundationdb/backups.html</a></p>",
        "post_number": 17,
        "post_type": 1,
        "posts_count": 17,
        "updated_at": "2019-12-17T19:51:11.604Z",
        "reply_count": 0,
        "reply_to_post_number": 16,
        "quote_count": 0,
        "incoming_link_count": 1,
        "reads": 33,
        "readers_count": 32,
        "score": 11.6,
        "yours": false,
        "topic_id": 1458,
        "topic_slug": "regarding-issue-faced-while-running-fdbbackup",
        "display_username": "John Brownlee",
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_bg_color": null,
        "flair_color": null,
        "flair_group_id": null,
        "badges_granted": [],
        "version": 1,
        "can_edit": false,
        "can_delete": false,
        "can_recover": false,
        "can_see_hidden_post": false,
        "can_wiki": false,
        "link_counts": [
          {
            "url": "https://apple.github.io/foundationdb/backups.html",
            "internal": false,
            "reflection": false,
            "title": "Backup, Restore, and Replication for Disaster Recovery \u2014 FoundationDB 6.2",
            "clicks": 30
          }
        ],
        "read": true,
        "user_title": null,
        "reply_to_user": {
          "id": 700,
          "username": "viveksinghggits",
          "name": "Vivek Singh",
          "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png"
        },
        "bookmarked": false,
        "actions_summary": [],
        "moderator": false,
        "admin": true,
        "staff": true,
        "user_id": 3,
        "hidden": false,
        "trust_level": 4,
        "deleted_at": null,
        "user_deleted": false,
        "edit_reason": null,
        "can_view_edit_history": true,
        "wiki": false,
        "post_url": "/t/regarding-issue-faced-while-running-fdbbackup/1458/17",
        "can_accept_answer": false,
        "can_unaccept_answer": false,
        "accepted_answer": false,
        "topic_accepted_answer": null
      }
    ],
    "stream": [
      4411,
      4417,
      5828,
      5830,
      5831,
      5832,
      5835,
      5837,
      5848,
      5851,
      5852,
      5855,
      5857,
      5859,
      5860,
      5861,
      5863
    ]
  },
  "timeline_lookup": [
    [
      1,
      2318
    ],
    [
      3,
      2136
    ],
    [
      9,
      2135
    ]
  ],
  "suggested_topics": [],
  "tags": [],
  "tags_descriptions": {},
  "fancy_title": "Regarding issue faced while running fdbbackup",
  "id": 1458,
  "title": "Regarding issue faced while running fdbbackup",
  "posts_count": 17,
  "created_at": "2019-06-17T12:33:04.574Z",
  "views": 2007,
  "reply_count": 14,
  "like_count": 1,
  "last_posted_at": "2019-12-17T19:51:11.604Z",
  "visible": true,
  "closed": false,
  "archived": false,
  "has_summary": false,
  "archetype": "regular",
  "slug": "regarding-issue-faced-while-running-fdbbackup",
  "category_id": 7,
  "word_count": 1381,
  "deleted_at": null,
  "user_id": 587,
  "featured_link": null,
  "pinned_globally": false,
  "pinned_at": null,
  "pinned_until": null,
  "image_url": null,
  "slow_mode_seconds": 0,
  "draft": null,
  "draft_key": "topic_1458",
  "draft_sequence": null,
  "unpinned": null,
  "pinned": false,
  "current_post_number": 1,
  "highest_post_number": 17,
  "deleted_by": null,
  "actions_summary": [
    {
      "id": 4,
      "count": 0,
      "hidden": false,
      "can_act": false
    },
    {
      "id": 8,
      "count": 0,
      "hidden": false,
      "can_act": false
    },
    {
      "id": 10,
      "count": 0,
      "hidden": false,
      "can_act": false
    },
    {
      "id": 7,
      "count": 0,
      "hidden": false,
      "can_act": false
    }
  ],
  "chunk_size": 20,
  "bookmarked": false,
  "topic_timer": null,
  "message_bus_last_id": 0,
  "participant_count": 5,
  "show_read_indicator": false,
  "thumbnails": null,
  "slow_mode_enabled_until": null,
  "tags_disable_ads": false,
  "related_topics": [
    {
      "fancy_title": "Backing up the database using `fdbbackup` command returns in an error",
      "id": 1844,
      "title": "Backing up the database using `fdbbackup` command returns in an error",
      "slug": "backing-up-the-database-using-fdbbackup-command-returns-in-an-error",
      "posts_count": 2,
      "reply_count": 0,
      "highest_post_number": 2,
      "image_url": null,
      "created_at": "2019-12-16T10:49:12.154Z",
      "last_posted_at": "2019-12-16T13:54:49.984Z",
      "bumped": true,
      "bumped_at": "2019-12-16T13:54:49.984Z",
      "archetype": "regular",
      "unseen": false,
      "pinned": false,
      "unpinned": null,
      "visible": true,
      "closed": false,
      "archived": false,
      "bookmarked": null,
      "liked": null,
      "tags": [],
      "tags_descriptions": {},
      "like_count": 0,
      "views": 1069,
      "category_id": 7,
      "featured_link": null,
      "has_accepted_answer": false,
      "posters": [
        {
          "extras": "latest single",
          "description": "Original Poster, Most Recent Poster",
          "user": {
            "id": 700,
            "username": "viveksinghggits",
            "name": "Vivek Singh",
            "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
            "trust_level": 1
          }
        }
      ]
    },
    {
      "fancy_title": "Backup /restore fdb",
      "id": 1637,
      "title": "Backup /restore fdb",
      "slug": "backup-restore-fdb",
      "posts_count": 22,
      "reply_count": 17,
      "highest_post_number": 22,
      "image_url": null,
      "created_at": "2019-09-19T17:04:18.878Z",
      "last_posted_at": "2019-10-05T04:45:51.663Z",
      "bumped": true,
      "bumped_at": "2019-10-05T04:45:51.663Z",
      "archetype": "regular",
      "unseen": false,
      "pinned": false,
      "unpinned": null,
      "visible": true,
      "closed": false,
      "archived": false,
      "bookmarked": null,
      "liked": null,
      "tags": [],
      "tags_descriptions": {},
      "like_count": 0,
      "views": 2729,
      "category_id": 7,
      "featured_link": null,
      "has_accepted_answer": false,
      "posters": [
        {
          "extras": null,
          "description": "Original Poster",
          "user": {
            "id": 604,
            "username": "Minaxi",
            "name": "meenakshi subramanyam",
            "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/m/2bfe46/{size}.png",
            "trust_level": 1
          }
        },
        {
          "extras": null,
          "description": "Frequent Poster",
          "user": {
            "id": 337,
            "username": "mengxu",
            "name": "Meng Xu",
            "avatar_template": "/user_avatar/forums.foundationdb.org/mengxu/{size}/893_2.png",
            "trust_level": 2
          }
        },
        {
          "extras": null,
          "description": "Frequent Poster",
          "user": {
            "id": 7,
            "username": "SteavedHams",
            "name": "Steve Atherton",
            "avatar_template": "/user_avatar/forums.foundationdb.org/steavedhams/{size}/18_2.png",
            "trust_level": 4
          }
        },
        {
          "extras": "latest",
          "description": "Most Recent Poster",
          "user": {
            "id": 13,
            "username": "alexmiller",
            "name": "Alex Miller",
            "avatar_template": "/user_avatar/forums.foundationdb.org/alexmiller/{size}/326_2.png",
            "trust_level": 4
          }
        }
      ]
    },
    {
      "fancy_title": "Backup_agent configuration for running fdbrestore",
      "id": 2870,
      "title": "Backup_agent configuration for running fdbrestore",
      "slug": "backup-agent-configuration-for-running-fdbrestore",
      "posts_count": 5,
      "reply_count": 2,
      "highest_post_number": 5,
      "image_url": null,
      "created_at": "2021-08-30T11:24:43.726Z",
      "last_posted_at": "2021-09-02T18:32:12.397Z",
      "bumped": true,
      "bumped_at": "2021-09-02T18:32:12.397Z",
      "archetype": "regular",
      "unseen": false,
      "pinned": false,
      "unpinned": null,
      "visible": true,
      "closed": false,
      "archived": false,
      "bookmarked": null,
      "liked": null,
      "tags": [],
      "tags_descriptions": {},
      "like_count": 0,
      "views": 824,
      "category_id": 7,
      "featured_link": null,
      "has_accepted_answer": false,
      "posters": [
        {
          "extras": null,
          "description": "Original Poster",
          "user": {
            "id": 994,
            "username": "maxim",
            "name": "maxim ",
            "avatar_template": "/user_avatar/forums.foundationdb.org/maxim/{size}/1211_2.png",
            "trust_level": 1
          }
        },
        {
          "extras": "latest",
          "description": "Most Recent Poster",
          "user": {
            "id": 12,
            "username": "ajbeamon",
            "name": "A.J. Beamon",
            "avatar_template": "/user_avatar/forums.foundationdb.org/ajbeamon/{size}/13_2.png",
            "admin": true,
            "trust_level": 4
          }
        }
      ]
    },
    {
      "fancy_title": "Running backup_agent alone",
      "id": 2107,
      "title": "Running backup_agent alone",
      "slug": "running-backup-agent-alone",
      "posts_count": 2,
      "reply_count": 0,
      "highest_post_number": 2,
      "image_url": null,
      "created_at": "2020-05-07T14:07:10.601Z",
      "last_posted_at": "2020-05-07T18:34:58.880Z",
      "bumped": true,
      "bumped_at": "2020-05-07T18:34:58.880Z",
      "archetype": "regular",
      "unseen": false,
      "pinned": false,
      "unpinned": null,
      "visible": true,
      "closed": false,
      "archived": false,
      "bookmarked": null,
      "liked": null,
      "tags": [],
      "tags_descriptions": {},
      "like_count": 1,
      "views": 296,
      "category_id": 7,
      "featured_link": null,
      "has_accepted_answer": false,
      "posters": [
        {
          "extras": null,
          "description": "Original Poster",
          "user": {
            "id": 81,
            "username": "ryanworl",
            "name": "Ryan Worl",
            "avatar_template": "/user_avatar/forums.foundationdb.org/ryanworl/{size}/440_2.png",
            "trust_level": 3
          }
        },
        {
          "extras": "latest",
          "description": "Most Recent Poster",
          "user": {
            "id": 7,
            "username": "SteavedHams",
            "name": "Steve Atherton",
            "avatar_template": "/user_avatar/forums.foundationdb.org/steavedhams/{size}/18_2.png",
            "trust_level": 4
          }
        }
      ]
    },
    {
      "fancy_title": "`fdbbackup` process not running after created by FoundationDBBackup",
      "id": 3341,
      "title": "`fdbbackup` process not running after created by FoundationDBBackup",
      "slug": "fdbbackup-process-not-running-after-created-by-foundationdbbackup",
      "posts_count": 7,
      "reply_count": 3,
      "highest_post_number": 7,
      "image_url": null,
      "created_at": "2022-05-27T03:06:01.948Z",
      "last_posted_at": "2022-05-31T06:54:05.607Z",
      "bumped": true,
      "bumped_at": "2022-05-31T06:54:05.607Z",
      "archetype": "regular",
      "unseen": false,
      "pinned": false,
      "unpinned": null,
      "visible": true,
      "closed": false,
      "archived": false,
      "bookmarked": null,
      "liked": null,
      "tags": [],
      "tags_descriptions": {},
      "like_count": 0,
      "views": 685,
      "category_id": 16,
      "featured_link": null,
      "has_accepted_answer": false,
      "posters": [
        {
          "extras": "latest",
          "description": "Original Poster, Most Recent Poster",
          "user": {
            "id": 970,
            "username": "mason",
            "name": "mason",
            "avatar_template": "/user_avatar/forums.foundationdb.org/mason/{size}/1145_2.png",
            "trust_level": 2
          }
        },
        {
          "extras": null,
          "description": "Frequent Poster",
          "user": {
            "id": 849,
            "username": "johscheuer",
            "name": "Johannes Scheuermann",
            "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/j/f475e1/{size}.png",
            "trust_level": 2
          }
        },
        {
          "extras": null,
          "description": "Frequent Poster",
          "user": {
            "id": 3,
            "username": "john_brownlee",
            "name": "John Brownlee",
            "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
            "admin": true,
            "trust_level": 4
          }
        }
      ]
    }
  ],
  "summarizable": false,
  "can_vote": false,
  "vote_count": 0,
  "user_voted": false,
  "discourse_zendesk_plugin_zendesk_id": null,
  "discourse_zendesk_plugin_zendesk_url": "https://your-url.zendesk.com/agent/tickets/",
  "details": {
    "can_edit": false,
    "notification_level": 1,
    "participants": [
      {
        "id": 700,
        "username": "viveksinghggits",
        "name": "Vivek Singh",
        "avatar_template": "/user_avatar/forums.foundationdb.org/viveksinghggits/{size}/737_2.png",
        "post_count": 8,
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_color": null,
        "flair_bg_color": null,
        "flair_group_id": null,
        "trust_level": 1
      },
      {
        "id": 3,
        "username": "john_brownlee",
        "name": "John Brownlee",
        "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png",
        "post_count": 5,
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_color": null,
        "flair_bg_color": null,
        "flair_group_id": null,
        "admin": true,
        "trust_level": 4
      },
      {
        "id": 12,
        "username": "ajbeamon",
        "name": "A.J. Beamon",
        "avatar_template": "/user_avatar/forums.foundationdb.org/ajbeamon/{size}/13_2.png",
        "post_count": 2,
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_color": null,
        "flair_bg_color": null,
        "flair_group_id": null,
        "admin": true,
        "trust_level": 4
      },
      {
        "id": 587,
        "username": "pragawaran",
        "name": "",
        "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/p/f07891/{size}.png",
        "post_count": 1,
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_color": null,
        "flair_bg_color": null,
        "flair_group_id": null,
        "trust_level": 1
      },
      {
        "id": 166,
        "username": "gaurav",
        "name": "gaurav",
        "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/g/b487fb/{size}.png",
        "post_count": 1,
        "primary_group_name": null,
        "flair_name": null,
        "flair_url": null,
        "flair_color": null,
        "flair_bg_color": null,
        "flair_group_id": null,
        "trust_level": 3
      }
    ],
    "created_by": {
      "id": 587,
      "username": "pragawaran",
      "name": "",
      "avatar_template": "https://avatars.discourse-cdn.com/v4/letter/p/f07891/{size}.png"
    },
    "last_poster": {
      "id": 3,
      "username": "john_brownlee",
      "name": "John Brownlee",
      "avatar_template": "/user_avatar/forums.foundationdb.org/john_brownlee/{size}/22_2.png"
    },
    "links": [
      {
        "url": "https://apple.github.io/foundationdb/backups.html",
        "title": "Backup, Restore, and Replication for Disaster Recovery \u2014 FoundationDB 6.2",
        "internal": false,
        "attachment": false,
        "reflection": false,
        "clicks": 30,
        "user_id": 3,
        "domain": "apple.github.io",
        "root_domain": "apple.github.io"
      },
      {
        "url": "https://apple.github.io/foundationdb/backups.html?highlight=backup#backup-agent-command-line-tool",
        "title": "Backup, Restore, and Replication for Disaster Recovery \u2014 FoundationDB 6.1",
        "internal": false,
        "attachment": false,
        "reflection": false,
        "clicks": 14,
        "user_id": 12,
        "domain": "apple.github.io",
        "root_domain": "apple.github.io"
      },
      {
        "url": "https://github.com/foundationdb/fdb-kubernetes-operator",
        "title": "GitHub - FoundationDB/fdb-kubernetes-operator: A kubernetes operator for FoundationDB",
        "internal": false,
        "attachment": false,
        "reflection": false,
        "clicks": 3,
        "user_id": 700,
        "domain": "github.com",
        "root_domain": "github.com"
      },
      {
        "url": "https://forums.foundationdb.org/t/backing-up-the-database-using-fdbbackup-command-returns-in-an-error/1844/2",
        "title": "Backing up the database using `fdbbackup` command returns in an error",
        "internal": true,
        "attachment": false,
        "reflection": true,
        "clicks": 1,
        "user_id": 700,
        "domain": "forums.foundationdb.org",
        "root_domain": "foundationdb.org"
      }
    ]
  },
  "bookmarks": []
}